### PR TITLE
Add image-to-text pipeline

### PR DESCRIPTION
**Pull request by mjh1**
Adding a new image-to-text pipeline, tested locally with the `Salesforce/blip-image-captioning-large` model.
Tested sending requests direct to runner as well as running gateway and worker/orch locally.

### CODE REVIEW

File: runner/gateway.openapi.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ openapi: 3.1.0
 info:
   title: Livepeer AI Runner
   description: An application to run AI pipelines
-  version: ''
+  version: v0.7.0
```
Comment (mjh1) [inline runner/gateway.openapi.yaml:5]: Not sure why these versions have been set and whether that's ok

Comment (rickstaa) [reply runner/gateway.openapi.yaml:5]: @mjh1 thanks for the pull request. Yea the version is not really used anymore now we use speakeasy. The version was accidentally removed during the LLM merge.

**Comment by mjh1** [comment]
Forgot I still need to add the worker code.. hang tight

**Comment by mjh1** [comment]
@rickstaa ready for a review now 🙏

**Comment by rickstaa** [comment]
@mjh1 great 🚀! I asked @pschroedl to review it.

**Comment by mjh1** [comment]
@pschroedl Please could I get a review? Thanks 🙏

File: runner/app/routes/image_to_text.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            content=http_error("File size exceeds limit"),
+        )
+
+    image = Image.open(image.file).convert("RGB")
+    try:
+        return ImageToTextResponse(text=pipeline(prompt=prompt, image=image))
+    except Exception as e:
+        logger.error(f"ImageToTextPipeline error: {e}")
+        logger.exception(e)
+        return JSONResponse(
```
Comment (gioelecerati) [inline runner/app/routes/image_to_text.py:95]: I think you could use the new exception handler in util.py here, similar to other pipelines, if you rebase on master
Also, on all the other pipelines a InferenceError was added, which can go in this pipeline too

**Review by gioelecerati — APPROVED** [review]

File: runner/app/routes/image_to_text.py (hunk 2 above)
Comment (mjh1) [reply runner/app/routes/image_to_text.py:95]: ah good point thanks

**Review by rickstaa — APPROVED** [review]
@mjh1 very clean pull request! Also amazing model. Thanks for the pull request 🚀. Feel free to merge 👍🏻. Will merge look at the go-livepeer tomorrow.

**Pull request merged**